xctool/xctool.xcconfig

```diff
@@ -28,7 +28,7 @@ XCTOOL_VENDOR_DIR = $(SRCROOT)/../Vendor
 
 // xctool will weak link following frameworks, and then configure the necessary
 // DYLD paths on startup to make sure they can be loaded.
-OTHER_LDFLAGS = -weak_framework DVTFoundation -weak_framework DVTiPhoneSimulatorRemoteClient -weak_framework CoreSimulator -weak_framework XCTest
+OTHER_LDFLAGS = -weak_framework DVTFoundation -weak_framework CoreSimulator -weak_framework XCTest
```

Comment:
Xcode 13 doesn't contain `DVTiPhoneSimulatorRemoteClient.framework` anymore. On Xcode 12.5, this framework is at `Contents/SharedFrameworks`.